Promise.resolve(3).then(result => console.log(result)); 
Promise.reject("Boo!").catch(echec => console.log(echec)); 
